stop_daemons.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# Stop all daemons
+
+sudo killall maap_daemon
```

Comment:
For possible future improvement:  Although killall is very convenient, it's not always provided in minimal OS configurations. I don't think there's a clear "best way" to do this across all bash-supporting systems, but there are probably a couple of ways that could be found in some cross-platform start/stop script implementations around.